extensions/sf-symbols-search/src/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { ActionPanel, CopyToClipboardAction, PasteAction, getPreferenceValues, List } from "@raycast/api";
+import fs from "node:fs";
+
+interface Preferences {
+  primaryAction: string;
+}
+
+export default function Command() {
+  const prefs: Preferences = getPreferenceValues();
+  const symbols: { name: string; symbol: string; categories: string[] }[] = JSON.parse(
+    fs.readFileSync(`.${__dirname}/assets/symbols.json`, { encoding: "utf8" })
```

Comment:
instead of using `__dirname` (which is confusing because the `__dirname` is different between the source and the bundle) , you can use `environment.assetsPath` (https://developers.raycast.com/api-reference/environment#environment)
```suggestion
    fs.readFileSync(`${environment.assetsPath}/symbols.json`, { encoding: "utf8" })
```

Reply:
Thanks for catching this @mathieudutour! Just pushed your suggested change.